assets/html/custombar.html

```diff
@@ -1,98 +1,32 @@
 <section id="customBarSectionMain">
-  <div class="main-custombar-container">
-    <div class="main-bag-container">
-      <div class="main-letters-container">
-        <div class="letters-wrapper">
-          <div class="bag-letter">
-            <div class="bag-letter-img-container">
-              <img src="" alt="" id="bagLetter1">
-            </div>
-          </div>
-          <div class="bag-letter">
-            <div class="bag-letter-img-container">
-              <img src="" alt="" id="bagLetter2">
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-    <div class="main-customization-container">
-      <div class="letters-input-wrapper">
-        <input type="text" id="letters-input" maxlength="2" pattern="[A-Z]" placeholder="Choose">
-      </div>
-      <div class="letters-customization-wrapper">
-        <div class="custom-letter-container">
-          <div class="tab-letter-border-buffer"></div>
-          <div class="tab-letter-wrapper active">
-            <div id="tab-letter1" class="tab-letter-color-letter active">
-            </div>
-          </div>
-          <div class="tab-letter-wrapper" id="tab-letter-wrapper">
-            <div id="tab-letter2" class="tab-letter-color-letter active">
-            </div>
-          </div>
-          <div class="tab-letter-border-buffer"></div>
-        </div>
+  <div class="bag">
+    <div class="letters js-letters"></div>
```

Comment:
so I moved away from that `elements` object, which was a little experimental, and went back to something I have used for years.

if you prefix classes that jquery depends on with `js-`, then a future developer is unlikely to move things around or delete things without also modifying the js.